### ISSUE TITLE
does it work on xcode6 & ios8.2 now?

### ISSUE THREAD

**Issue by sunuslee** [CyberAgent/iOS-NBUCore#5]
hi, i just use this simple code to test, but get no warnings:
my baseSDK is ios8.2 and deployment target is 6.0..

    UIView *testView = [[UIView alloc] initWithFrame:self.window.bounds];
    [testView setLayoutMargins:UIEdgeInsetsZero];

i think the setLayoutMagins , should be introduced in ios8. 
    
    @property (nonatomic) BOOL preservesSuperviewLayoutMargins NS_AVAILABLE_IOS(8_0);

is there something i need to be noticed?
thanks!

**Comment by rivera-ernesto** [comment]
You make [this `#define`](https://github.com/CyberAgent/iOS-NBUCore/blob/master/Source/NBUAvailability.h#L31) just before importing `NBUAvailability.h`.

We may need to add entries for 8.2 and 8.3 in the bottom of that file though.

**Comment by sunuslee** [comment]
i tried, it didn't give me any warnings. 
are you sure this work?

also, i am putting another some test program here, still, strange.

could you please clone that repo and tell me what's going on? thanks!

https://github.com/sunuslee/macapi-tester

**Comment by sunuslee** [comment]
it turn out to be the same situation when i add

    #if __IPHONE_OS_VERSION_SOFT_MAX_REQUIRED < __IPHONE_8_2
        #undef __AVAILABILITY_INTERNAL__IPHONE_8_2
        #define __AVAILABILITY_INTERNAL__IPHONE_8_2 __NBU_AVAILABILITY_STARTING("8.2")
        #define __NBU_APICHECK_8_2(_ios)            __NBU_AVAILABILITY_STARTING("8.2")
    #else
        #define __NBU_APICHECK_8_2(_ios)            CF_AVAILABLE_IOS(_ios)
    #endif

 to this file, then, in my class(AppDelegate)

add a property (same as UIView)
    @property (nonatomic) UIEdgeInsets layoutMargins NS_AVAILABLE_IOS(8_0);

then,

    [self setLayoutMargins:UIEdgeInsetsZero];

gets the right warning.

![screen shot 2015-04-03 at 11 51 58 am](https://cloud.githubusercontent.com/assets/584417/6977792/f689d402-d9f7-11e4-99a2-2f24fe482bad.png)

but no luck for

    UIView *testView = [[UIView alloc] initWithFrame:self.window.bounds];
    [testView setLayoutMargins:UIEdgeInsetsZero];

**Comment by rivera-ernesto** [comment]
Did you try adding the entries for 8.1 ~ .3 to `NBUAvailability.h`?

**Comment by sunuslee** [comment]
yeah, it turned out that, i have to set enable_module to NO, so that those macros can work.

**Comment by rivera-ernesto** [comment]
Nice. Could you pull request your changes?

**Comment by rivera-ernesto** [comment]
I updated the library and set `enable_module` to `NO` but still macros don't show up.
